### DISCUSSION TITLE
Add heatmap and power graph based on location

### DISCUSSION THREAD

**Discussion by qkhaleghi** [VREMSoftwareDevelopment/WiFiAnalyzer#469] [Ideas]
It is very interesting and practical if it is possible to measure the strength of each signal based on the location of the device in order to create a complete map of the devices in a house or a larger office.
https://user-images.githubusercontent.com/3512839/71321256-12212a80-24b7-11ea-85b9-034eb4d128f5.png

**Comment by VREMSoftwareDevelopment** [comment]
This is a feature request, not a discussion
